haproxy_stats.sh

```diff
@@ -1,25 +1,16 @@
 #!/bin/bash
 set -o pipefail
 
-if [[ "$1" = /* ]]
-then
-  HAPROXY_SOCKET="$1"
-  shift 1
-fi
+[ "$#" -gt 3 ] && HAPROXY_SOCK="$(echo $1 | tr -d '\040\011\012\015')" && shift 1
+
+# Define variables and perform checks
+CONFIG_FILE="$(dirname $(readlink -f "$0"))/haproxy_config"
+. $CONFIG_FILE
```

Comment:
:+1: 
except `HAPROXY_SOCKEt` -> `HAPROXY_SOCK` so var will be similar for every block